.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+name: CI
+on: [push, pull_request]
+env:
+  CI: true
+  SCALA_VERSION: 2.13.1
+jobs:
+  validate:
+    name: Scala ${{ matrix.scala }}, Java ${{ matrix.java }}
+    runs-on: ${{matrix.systems}} 
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [adopt@1.8, adopt@1.11, adopt@1.14,graalvm@]
+        scala: [2.12.11, 2.13.1]
+        #TODO: windows-latest likely requires another [job](https://github.com/olafurpg/setup-scala)
+        systems: [ubuntu-latest, macos-latest, windows-latest]
+    env:
+      SCALA_VERSION: ${{ matrix.scala }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v7
+        with:
+          java-version: ${{ matrix.java }}
+      # TODO: should we remove?
+      - name: Cache Coursier
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/coursier
+          key: sbt-coursier-cache
+      # TODO: should we remove?     
```

Comment:
I think this one is still right.

Reply:
I'm lazy and don't know what it's doing for me. But that's fine.

Reply:
This loads all the dependencies from a GitHub cache instead of from Maven Central.  The build would work fine without it.  It would just be a little slower, and probably cost GitHub a little more money. :smile:

Reply:
Ok, then I should remove the comment. I will read the readme next time. I'm slightly motivated to get things out more quickly sometimes.